lib/dry/system/booter.rb

```diff
@@ -69,6 +69,15 @@ def finalize!
         freeze
       end
 
+      # @api private
+      def shutdown
+        components.each do |component|
+          stop(component)
+        rescue ComponentNotStartedError
```

Comment:
We could just iterate over `booted` and don't have to handle `ComponentNotStartedError` exception. Whay not?

Reply:
I also find it strange, that booter.stop(<component>) doesn't upload a component from `booted` registry, so it doesn't allow to start the same component twice. Or it is by design? If yes, why? @solnic @GustavoCaso

Reply:
I doubt this is by design, I think it's an oversight and something we should fix.

Reply:
It should be allowed to start something multiple times, so it's just not finished, that's all.

Reply:
@v-kolesnikov nice catch about using `booted` instead of components. It would also fix CI since we wouldn't need the `rescue` part @flash-gordon pointed out ❤️